src/cli/backup/backup.go

```diff
@@ -288,7 +296,7 @@ func genericListCommand(cmd *cobra.Command, bID string, service path.ServiceType
 		}
 
 		b.Print(ctx)
-		fe.PrintItems(ctx, listFailedItems != "show", listSkippedItems != "show")
+		fe.PrintItems(ctx, listFailedItems != "show", listSkippedItems != "show", listRecoveredErrors != "show")
```

Comment:
do we need to normalize text here so we don't run into case issues?

Reply:
oh, yeah, good call